src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -154,43 +72,21 @@ const CustomFieldInputSingleChoice = forwardRef(function CustomFieldInputSingleC
 
   return (
     <div ref={wrapperRef} className={props.className}>
```

Comment:
Looks like we don't use `wrapperRef` anymore. We might even be able to throw away the className too.

Reply:
I can see removing the `wrapperRef`, but I feel like we should maintain the `className` API to provide for style overrides in the future, and to keep parity with the other components.